source/index.html.md.erb

```diff
@@ -562,6 +564,8 @@ WRITE_SHEETS | Insert and modify sheet data, including attachments, discussions,
 Your app must implement a 3-legged OAuth flow to retrieve an access token that can be used to access Smartsheet data on behalf of an end user. 
 The following sub-sections describe next steps.
 
+<aside class="notice">App registration and OAuth flow require HTTPS.</aside>
```

Comment:
(I'm assuming this is in response to the email or support case) As written, this isn't actionable. Needs to be documented as a requirement on the redirect_uri in the token endpoints.

Reply:
Done